docs/features/service-bus.md

```diff
@@ -0,0 +1,59 @@
+---
+title: "Azure Service Bus Features"
+layout: default
+---
+
```

Comment:
Can we add installation section similar to this please?

https://github.com/arcus-azure/arcus.messaging/blob/master/docs/features/tcp-health-probe.md#L10

Reply:
👍